src/main/java/mate/academy/bookstore/service/impl/UserServiceImpl.java

```diff
@@ -33,6 +40,13 @@ public UserResponseDto register(UserRegistrationRequestDto request)
         user.setShippingAddress(request.getShippingAddress());
         user.setPassword(passwordEncoder.encode(request.getPassword()));
         user.setEmail(request.getEmail());
+        Role roleUser =
+                roleRepository.findById(2L).orElseThrow(

```

Comment:
It is better to use the name of the Role
```suggestion
                roleRepository.findByName(Role.RoleName.USER).orElseThrow(
```